misctools/fetch.zig

```diff
@@ -187,7 +187,7 @@ pub fn main() anyerror!void {
     var ctx = try default_allocator.create(HTTP.HTTPChannelContext);
     ctx.* = .{
         .channel = channel,
-        .http = try HTTP.AsyncHTTP.init(default_allocator, args.method, args.url, args.headers, args.headers_buf, response_body_string, args.body, 0, HTTP.FetchRedirect.follow),
+        .http = try HTTP.AsyncHTTP.init(default_allocator, args.method, args.url, args.headers, args.headers_buf, response_body_string, args.body, 0, HTTP.FetchRedirect.follow, false),
```

Comment:
can we make this an options struct

hard to know what `false` here means

Reply:
Sure, maybe adding the min buffer size for the streaming too.